code/src/containers/tables/products-table/ProductsTable.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from "@testing-library/react";
+
+import ProductsTable from "@/containers/tables/products-table/ProductsTable";
+import {
+  mockProducts,
+  productsTableColumns
+} from "@/containers/tables/products-table/ProductsTable.constants";
+
+describe("Test ProductsTable", () => {
+  test("Should be rendered correctly", () => {
+    const { container } = render(<ProductsTable products={mockProducts} />);
+
+    productsTableColumns
+      .filter((item) => item)
+      .forEach((columnTitle) => {
+        const columnElement = screen.getByText(columnTitle);
+        expect(columnElement).toBeInTheDocument();
+      });
+
+    const tableCells = screen.getAllByRole("cell");
+
+    const bodyElement = container.querySelector(".products-table__body");
+    const containerElement = container.querySelector(".products-table");
+
+    expect(tableCells.length).toBe(7 * 2);
```

Comment:
Why not 14?

Reply:
I think that it is more understandable - 2 rows with 7 cells in each

Reply:
I suggest writing small comment for it

Reply:
Is it really needed there? I guess developer can understand this from variable names like `tableCells`